app/src/main/java/com/framgia/soundclound/util/Constant.java

```diff
@@ -1,6 +1,11 @@
 package com.framgia.soundclound.util;
 
 
+import android.Manifest;
+
+import java.util.ArrayList;
```

Comment:
ERROR: Unused import: java.util.ArrayList. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports